### ISSUE TITLE
Not working with Galacticraft

### ISSUE THREAD

**Issue by Romchest** [Shovinus/ChopDownUpdated#11]
ChopDown+Updated+(1.12.2+)-1.0.10

GalacticraftCore-1.12.2-4.0.1.177
MicdoodleCore-1.12.2-4.0.1.177
Galacticraft-Planets-1.12.2-4.0.1.177

When I break lower tree block, nothing happens.
There is no erros in log file
It only works without Galacticraft

**Comment by Shovinus** [comment]
Going to guess that galactic craft messes with the provided player instance for some reason, thus my code thinks that the player is a fake player and does not do the chop, will look later

On 27 May 2018 10:23, Zygus42 <notifications@github.com> wrote:

I can confirm this.

v1.0.9 works fine, but when I updated to 1.0.10 it stopped working, and I eventually tracked it to Galacticraft as well.

1.0.10 works if Galacticraft is not part of the modpack. Tested with Galacticraft v175 and v177.

And makes no sense at all really :) Good luck finding the problem.
